test/Microsoft.AspNetCore.Server.Kestrel.Performance/RequestParsingDataBenchmark.cs

```diff
@@ -8,7 +8,7 @@
 namespace Microsoft.AspNetCore.Server.Kestrel.Performance
 {
     [Config(typeof(CoreConfig))]
-    public class RequestParsingData
+    public class RequestParsingDataBenchmark
```

Comment:
This class shouldn't have the `Benchmark` suffix, since it's not a benchmark class.

You can remove the `Config` attribute from it too, it was a copy & paste mishap when I created the class (hadn't realized it till now).